3rdParty/V8/v7.1.302.28/src/base/bits.h

```diff
@@ -151,7 +151,7 @@ inline size_t RoundUpToPowerOfTwo(size_t value) {
   if (sizeof(size_t) == sizeof(uint64_t)) {
     return RoundUpToPowerOfTwo64(value);
   } else {
-    return RoundUpToPowerOfTwo32(value);
+    return RoundUpToPowerOfTwo32(static_cast<uint32_t>(value));
```

Comment:
Now there would probably be a warning if we would compile for 32 bit. Maybe the cleanest solution is to use normal overloading for uint32_t and uint64_t. What would even eliminate the check.

Reply:
fixed by using the ifdef found in the other place.